library/src/interactor_impl.cxx

```diff
@@ -363,29 +363,13 @@ class interactor_impl::internals
           vtkMath::Subtract(posV, v, posV);             /* pos -> pos2, keeps on camera plane */
         }
 
-        if (self->TransitionDuration > 0)
+        const auto update_camera = [&foc, &focV, &pos, &posV](camera& cam, double t)
```

Comment:
avoid single letter var :)

Reply:
[precedent](https://github.com/f3d-app/f3d/pull/491#discussion_r1029550176) :)

kinda stuck (again!) on the variable name here, `interpolant` would be the most descriptive but is kind of a drag. Could be `time` but is that confusing as it goes from `0` through `1` without relation to physical time? Any suggestions?

Reply:
`interpolant` is fine by me, `time` is too. maybe `timeRatio` ?